backend-config/namespace_config.go

```diff
@@ -168,7 +168,7 @@ func (nc *namespaceConfig) getFromAPI(ctx context.Context) (map[string]ConfigT,
 		} else {
 			workspace.ApplyReplaySources()
 		}
-
+		workspace.processAccountAssociations()
```

Comment:
We should only `processAccountAssociations` if `workspace != nil`, i.e. this is a modified workspace. In the other case, the previous config will contain an already processed workspace